main.go

```diff
@@ -31,38 +31,41 @@ import (
 	"os"
 )
 
+func checkErrorCodeAndExit(err error) {
```

Comment:
Can we move those to utilities or create even a separate package for those, if there is no suitable place?

Reply:
I'll move functions to utilities.